packages/backend/src/api/endpoints/InvitationApi.ts

```diff
@@ -8,6 +8,8 @@ type CreateParams = {
   emailAddress: string;
   redirectUrl?: string;
   publicMetadata?: UserPublicMetadata;
+  delay?: boolean;
```

Comment:
I don't see `delay` mentioned in the BAPI docs and also can't find it (quick glance) in the source code. Where would that be coming from?

Reply:
I don't see a `delay` param either but there is a `notify?: boolean` instead.
ref: https://clerk.com/docs/reference/backend-api/tag/Invitations#operation/CreateInvitation!path=notify&t=request